### ISSUE TITLE
Error: Doc factory failed: amp-video-docking: Cannot read property 'getAttribute' of undefined

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#24252]
```
Error: Doc factory failed: amp-video-docking: Cannot read property 'getAttribute' of undefined
at getAttribute (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/dom.js:690)
at isRTL (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/extensions/amp-video-docking/0.1/amp-video-docking.js:276)
at (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service.js:398)
at getServiceInternal (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service.js:168)
at (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/extensions-impl.js:346)
at (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/extensions-impl.js:407)
at forEach (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/extensions-impl.js:405)
```

go/ampe/CM2H9OvX1deHPQ
